src/main/java/seedu/address/model/person/exceptions/MeetingNotFoundException.java

```diff
@@ -0,0 +1,6 @@
+package seedu.address.model.person.exceptions;
+
+/**
+ * Signals that the operation is unable to find the specified meeting.
+ */
+public class MeetingNotFoundException extends RuntimeException {}
```

Comment:
should there constructor with a warning message similar to the other exceptions?
```suggestion
public class MeetingNotFoundException extends RuntimeException {
    public MeetingNotFoundException() {
        super("meeting not found");
    }
}
```